### PR TITLE
Change Codewind install link #165

### PR DESCRIPTION
**Pull request by micgibso**
Signed-off-by: micgibso <micgibso@uk.ibm.com>

### CODE REVIEW

**Comment by micgibso** [comment]
@tetchel Drafted, please review, see comment in https://github.com/eclipse/codewind/issues/165, thx!

File: docs/_documentations/mdt-vsc-getting-started.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ To install Codewind for VS Code, complete the following steps:
 
 1. Install [VS Code version 1.28 or later](https://code.visualstudio.com/download).
 2. Install [Docker](https://docs.docker.com/install/) 17.06 or later. If you use Linux, you also need to install Docker Compose.
-3. Install Codewind from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind), or by searching for *Codewind* in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+3. Install Codewind by clicking [this link](vscode:extension/IBM.codewind), or by searching for *Codewind* in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
```
Comment (tetchel) [inline docs/_documentations/mdt-vsc-getting-started.md]: can we keep the marketplace link, too?

Comment (micgibso) [reply docs/_documentations/mdt-vsc-getting-started.md]: Marketplace link added back in as the last option in the sentence.

**Comment by micgibso** [comment]
@tetchel Link to marketplace added back in, thx.

File: docs/_documentations/mdt-vsc-getting-started.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ To install Codewind for VS Code, complete the following steps:
 
 1. Install [VS Code version 1.28 or later](https://code.visualstudio.com/download).
 2. Install [Docker](https://docs.docker.com/install/) 17.06 or later. If you use Linux, you also need to install Docker Compose.
-3. Install Codewind from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind), or by searching for *Codewind* in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+3. Install Codewind by clicking [this link](vscode:extension/IBM.codewind), by searching for *Codewind* in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions), or by installing Codewind from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind).
```
Comment (tetchel) [inline docs/_documentations/mdt-vsc-getting-started.md]: small thing but can we reorder it, i think in terms of priority it should be this link > marketplace > search in extensions view

Comment (tetchel) [reply docs/_documentations/mdt-vsc-getting-started.md]: The link _is_ an 'install from marketplace', so maybe we can combine the two, something like

Install Codewind by clicking [this link](vscode:extension/IBM.codewind) which installs from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind), or by searching for *Codewind* in the VS Code Extensions view

Comment (micgibso) [reply docs/_documentations/mdt-vsc-getting-started.md]: @tetchel Sure thing, have combined the two now.

**Pull request merged**